### ISSUE TITLE
Playlist for the speaker

### ISSUE THREAD

**Issue by GrimPixel** [drwhut/tabletop-club#272]
**Is your feature request related to a problem? Please describe.**
I would like to create playlists and play random music from one of them.

**Comment by drwhut** [comment]
I agree, being able to shuffle the music would be nice, when playing with friends we often change the track manually every so often anyways. One thing I want to keep in mind though is if playlists are going to be implemented, they should be as intuitive to use as possible. As an example, I really don't like how Final Fantasy XIV's orchestrons work, it's almost a minigame in of itself to try and set up a playlist, and it's not obvious when a playlist is being shuffled vs being played in order.
